lib/web.mlx

```diff
@@ -188,6 +188,75 @@ let getting_started ~install_url () =
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " is letting your editor run the provided "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " script. The installer places this script at ~/.dune/bin/ocamllsp. The script is deliberately named the same as the "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " executable so existing editor configurations will probably still work with the Dune Developer Preview (the exception is Visual Studio Code - see below)."
+    </p>
+    <p class_="text-white mt-4">
+    "Running "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " in a project with a dune.lock directory (generated by running "
+    <span class_="font-mono text-block-p">"dune pkg lock"</span>
+    ") will download and install "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " in the current project's _build directory, and then run it (yes that means that for now running "
+    <span class_="font-mono text-block-p">"dune clean"</span>
+    " will uninstall "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    "). If "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " is run elsewhere (outside a Dune project, or inside a Dune project lacking a dune.lock directory) then it will attempt to fall back to the installation of "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " in the active Opam switch. This way you can continue to work on projects that manage their dependencies with Opam after installing the Dune Developer Preview without changing your editor configuration."
+    </p>
+    <h4 class_="mt-2.5">
+      "Visual Studio Code"
+    </h4>
+    <p class_="text-white">
+    "By default the "
+    <Link class_="hover:underline" href="https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform">"vscode-ocaml-platform"</Link>
+    " plugin for Visual Studio Code assumes that you'll be using Opam to install "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    ". To configure it to run the "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    "  script installed with the Dune Developer Preview by adding this to your settings.json file:"
+    </p>
+    <Code>
+"{"
+"  \"ocaml.sandbox\": {"
+"    \"kind\": \"custom\","
+"    \"template\": \"$prog $args\""
+"  }"
+"}"
+    </Code>
+    <h4 class_="mt-2.5">
+      "Neovim"
+    </h4>
+    <p class_="text-white">
+    "Recent versions of neovim have an LSP client built into them, so it just needs to be configured to run "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " inside Dune projects:"
+    </p>
+    <Code>
+"vim.api.nvim_create_autocmd('FileType', {"
+"  pattern = 'ocaml',"
+"  callback = function(args)"
+"    vim.lsp.start({"
+"      name = 'ocaml-lsp-server',"
+"      cmd = {'ocamllsp'},"
+"      root_dir = vim.fs.root(args.buf, {'dune-workspace', 'dune-project'}),"
+"    })"
```

Comment:
I would more expect people to use the `nvim-lsp-config` plugin as it is a recommanded way to use LSPs in Neovim. As a result, I would say it work out-of-the-box and don't require extra configuration.